config/prow/plugins.yaml

```diff
@@ -271,3 +271,8 @@ external_plugins:
     events:
       - issue_comment
       - pull_request
+  ocp-power-automation/ci-monitoring-automation:
+  - name: mergable
```

Comment:
Is that mergeable?

Reply:
Yes, it was a typo with the previous repo at L269 as well.
I think it has nothing to do with this plugin.
I shall get back.